body.tex

```diff
@@ -0,0 +1,440 @@
+
+\subsection{Other Related Artifacts}
+
+Together with the creation of the release candidate, other artifacts may need to be branched, and possibly have a first release candidate also:
+
+\begin{itemize}
+\item Documentation: to consolidate all information relevant for the new release. This includes the release note.
+As an example, for Science Pipelines, the package \textit{pipelines\_lsst\_io} need to be branched.
+\item  Environment definition packages, like for example packages containing conda environment files (YAML).
+\item  Distribution packages; still referring to the Science Pipelines, the package \textbf{lsst}  where \textit{newinstal.sh} is developed, need to be branched also.
+\item Test data packages.
+\item Other packages not tagged automatically, but required to be inline with the released software.
+\end{itemize}
+
+
+\subsection{Release Candidates Validation} \label{sec:rcvalidation}
+
+The release candidate needs to be validated.
+
+In an ideal case, a test campaign following a specific test plan should be conducted, demonstrating that the release candidate, and therefore the forthcoming release, is behaving as required and expected.
+
+Practically, the validation in many cases can be:
+
+\begin{itemize}
+\item installation/configuration of the binaries packages, or distribution image; usually done manually
+\item inspect of the installation, that all expected files and configuration are there
+\item execution of some demo package available case by case depending of the SW product
+\item try some use cases in order to prove that the release candidate is behaving properly
+\item ask downstream users to act as beta testers.
+\end{itemize}
+
+For Science Pipelines, a characterization report is produced, in order to document the release outputs.
+This is a new document created for each major release.
+
+
+\subsection{Resolving Problems}
+
+In case problems are found during the validation, a DM issue needs to be created in Jira.
+This issue shall follow the usual development process as described in the developer guide.
+The fix will be first implemented in a ticket branch, reviewed and merged to master.
+Once the fix has been proved to work, it can be backported to the release branch.
+
+The release branch will be created only for the packages to be fixed, and will follow the naming convention described in \ref{sec:branchnaming}.
+
+%Given a fix to backport, documented in DM-XXX Jira issue, and given a release branch bM.n.x,
+Backporting mechanism can be summarized as follows \footnote{This needs to be documented in the DevGuide}:
+
+
+\begin{itemize}
+\item Create the backporting Jira issue DM-XYZ with subject \textit{Backport issue DM-XXX on release branch bM.n.x}
+  \begin{itemize}
+  \item DM-XXX Jira issue has to be already in status reviewed or done, and the corresponding ticket branch already merged to master.
```

Comment:
DM-XXX doesn't have to have been completed in order to create the backporting issue, surely?

Reply:
I think that if DM-XXX is in status "Reviewed", it can be backported. However, if you prefer we can state that DM-XXX need to be in status "Done" before it can be backported.